tests/unit/metrics/export/test_value.py

```diff
@@ -135,3 +146,22 @@ def test_init(self):
         bucket = value_module.Bucket(1, self.exemplar)
         self.assertEqual(bucket.count, 1)
         self.assertEqual(bucket.exemplar, self.exemplar)
+
+
+class TestExplicit(unittest.TestCase):
+    def test_init(self):
+        bounds = [1, 2]
+        explicit = value_module.Explicit(bounds)
+        self.assertEqual(explicit.bounds, bounds)
+
+    def test_bad_init(self):
+        with self.assertRaises(ValueError):
+            value_module.Explicit(None)
+        with self.assertRaises(ValueError):
+            value_module.Explicit([])
+        with self.assertRaises(ValueError):
+            value_module.Explicit([0, 1, 2])
```

Comment:
Could you add a test for negative values?

Reply:
Added in 85b41d1.